platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -21,9 +21,9 @@ namespace mbgl {
 <% if (layoutProperties.length) { -%>
 <% for (const property of layoutProperties) { -%>
```

Comment:
I mistakenly thought the properties were alphabetically sorted to begin with, but you’re right that the properties are far from sorted in the style specification’s JSON objects. The [official documentation](https://www.mapbox.com/mapbox-gl-style-spec/) is generated using v8, just like our generated runtime styling files, so for example `icon-rotate` comes before `icon-padding`. @lucaswoj doesn’t know of any reason we need to keep the properties in that order, so alphabetizing the properties makes sense because it’ll make it easier to find them in our jazzy documentation.